### ISSUE TITLE
Add `<label>` for language dropdown in top-right corner

### ISSUE THREAD

**Issue by neel-a-mehta** [seattle-uat/civiform#1471]
The language dropdown in the top right corner shows the list of available languages and lets the user choose one. The problem is that we never say what that dropdown does, and so a screen reader would just read the active language without explaining what the dropdown is for:

![5gKzSxBVrZftJaz](https://user-images.githubusercontent.com/77128518/123181599-745b4280-d442-11eb-81f2-4f0e8f0947ce.png)

Instead, we should add a `<label>` for this `<select>`, [like in this guide](https://www.w3schools.com/tags/att_option_label.asp). We can use Tailwind's [`.sr-only` class](https://tailwindcss.com/docs/screen-readers) to hide it from people not using screen readers, though

This way, when the screen reader goes to the dropdown, it will read something like "Choose your language:" before reading off the list of languages. This will be much easier to understand.

## Original tester's report

> ACTUAL RESULT: Language popup button is labeled with default selected list item 'English' button.
> EXPECTED RESULT: Predefined label should be associated with the Language popup button Instead announcing the selected language as a button.
